FunctionalTableData/CellStyle.swift

```diff
@@ -9,18 +9,41 @@
 import Foundation
 import UIKit
 
+/// The actions property exposed on the CellConfigType represents possible events that will be executed based on the users interaction with that particular cell. Of note are the `selectionAction` and `previewingViewControllerAction`. The `selectionAction` is executed when the user taps on that particular cell. The main use case for this is present a new detail view controller or a modal (but is not constrained to these actions, these are just the common use cases). The `previewingViewControllerAction` is responsible for returning an instance of a UIViewController that will be shown when a user 3D-touches on a cell.
 public struct CellStyle {
 	public static var selectionColor: UIColor? = nil // TODO: make this not a static like this
+	/// The style to apply to the bottom separator in the cell.
+	///
+	/// Supported by UITableView only.
 	public var bottomSeparator: Separator.Style?
+	/// The style to apply to the top separator in the cell.
+	///
+	/// Supported by UITableView only.
 	public var topSeparator: Separator.Style?
+	/// The color of separator rows in the cell.
```

Comment:
Clarification: `lines`, not `rows` (`rows` can be confused with the cells themselves.)